frontends/ui/src/app/globals.css

```
@@ -312,3 +312,130 @@ body {
+}
+.answer-prose ul,
+.answer-prose ol {
+  margin: 0.5rem 0 0.85rem;
+  padding-left: 1.25rem;
+}
+.answer-prose li {
+  margin: 0.2rem 0;
+}
+
+/* ---- Status atom: one visual language for every agent-trace node ------------ */
+@keyframes dotBreathe {
```

Stylelint (17.14.0)

[error] Expected keyframe name "dotBreathe" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)